src/mozilla_taskgraph/config.py

```diff
@@ -0,0 +1,12 @@
+from taskgraph import config as tg
+from voluptuous import Optional, Required
+
+tg.graph_config_schema = tg.graph_config_schema.extend(
+    {
+        Optional("shipit"): {
```

Comment:
Food for thoughts: mozilla-beta currently supports 2 products (`firefox` and `devedition`). We'll likely have more once Fenix and Focus are migrated to the gecko repos. No immediate action needed but we'll likely have to support multiple products later on.